apps/web/src/pages/settings/SettingsPage.tsx

```diff
@@ -1,39 +1,11 @@
-import { Center, Container, Loader, Tabs } from '@mantine/core';
```

Comment:
basically this file was renamed to `SettingsPageOld` not sure why GH sees this as a changes